src/guides/config.md

```diff
@@ -91,6 +91,7 @@ Note that certain chain options affect the format and indexing of the chain data
 
 ### Pool Options
 
+- `agent`: Custom user agent (default: /hsd:version/). Must be not more than 255 characters long.
```

Comment:
Agent doesn't replace the full user agent. It adds what's provided after `/hsd:version/`

Example: `--agent bob-wallet:2.0.0` will make it `/hsd:5.0.1/bob-wallet:2.0.0/`

Reply:
Indeed, I know. How would you concisely rephrase this?  
It's not said that this setting replaces the default, so seems to be correct, just not "complete".

Reply:
Stealing it from bitcoin core: https://github.com/bitcoin/bitcoin/blob/40e1c4d4024b8ad35f2511b2e10bf80c5531dfde/src/init.cpp#L574

Reply:
tiny nit: should we clarify that the max length applies to the whole agent and not just the custom part?